src/main/java/com/bugsnag/android/Client.java

```diff
@@ -580,6 +585,11 @@ private void notify(final Error error, boolean blocking) {
             return;
         }
 
+        // Add a breadcrumb for this error occurring
+        HashMap<String, String> breadcrumbMetadata = new HashMap<String, String>();
+        breadcrumbMetadata.put("message", error.getExceptionMessage());
```

Comment:
Use `Collections.singletonMap` for a much more space-efficient implementation of a single-element `Map`